elements/pfe-cta/src/pfe-cta.js

```diff
@@ -13,42 +13,94 @@ class PfeCta extends PFElement {
     return "pfe-cta.html";
   }
 
+  get defaultStyle() {
+    return this.hasAttribute("priority") ? false : true;
+  }
+
+  static get observedAttributes() {
+    return [
+      "pfe-priority",
+      "pfe-color",
+      "pfe-alt",
+      "on"
+    ];
+  }
+
+  // Declare the type of this component
+  static get PfeType() {
+    return PFElement.PfeTypes.Content;
+  }
+
   constructor() {
     super(PfeCta);
+    this.cta = null;
+
     this._init = this._init.bind(this);
+    this._focusHandler = this._focusHandler.bind(this);
+    this._blurHandler = this._blurHandler.bind(this);
   }
 
   connectedCallback() {
     super.connectedCallback();
+
     this._slot = this.shadowRoot.querySelector("slot");
     this._slot.addEventListener("slotchange", this._init);
+
+    this._init();
+
+    // Get the lightDOM link if it exists
+    this.cta.addEventListener("focus", this._focusHandler);
+    this.cta.addEventListener("blur", this._blurHandler);
   }
 
   disconnectedCallback() {
     this._slot.removeEventListener("slotchange", this._init);
+
+    // Remove the focus state listeners
+    this.cta.removeEventListener("focus", this._focusHandler);
+    this.cta.removeEventListener("blur", this._blurHandler);
   }
 
   _init() {
     const firstChild = this.children[0];
 
-    if (!firstChild) {
-      console.warn(
-        `${
-          PfeCta.tag
-        }:The first child in the light DOM must be an anchor tag (<a>)`
-      );
-    } else if (firstChild && firstChild.tagName.toLowerCase() !== "a") {
+    if (!firstChild || (firstChild && !["A", "BUTTON", "INPUT"].includes(firstChild.tagName))) {
```

Comment:
@castastrophe this is throwing errors in IE11; when I revert it back to the original code it works. Can we back this out & make it a todo for later?

Reply:
Actually I can't replicate any errors.  I just pulled up the demo page in IE11 on BrowserStack and see no error messages in the console.

Reply:
I added a polyfill for includes because it's not supported in IE11 - let me know if that does not resolve the issue you are seeing.

Reply:
same, no errors anymore, thx!